astroquery/esasky/core.py

```diff
@@ -1230,6 +1237,18 @@ def _get_maps_for_mission(self, maps_table, mission, download_dir, cache, json,
 
         return maps
 
+    def _ends_with_fits_like_extentsion(self, name):
```

Comment:
Oh, what an arsenal of names for INTEGRAL files :)

Reply:
Hehe, I imagine INTEGRAL is somewhat standardized regarding the extensions, but I decided to put all the versions I have encountered over the years, just to be safe.